client/dashboard/sites/settings-site-visibility/privacy-form.tsx

```diff
@@ -211,7 +212,7 @@ export function PrivacyForm( { site, settings }: { site: Site; settings: SiteSet
 												<Button
 													variant="secondary"
 													href={ addQueryArgs( `/domains/add/${ site.slug }`, {
-														redirect_to: window.location.pathname,
+														redirect_to: redirectToDashboardLink( { supportBackport: true } ),
```

Comment:
I think we need to update the destination link as well. The button does not work in v2 🙈 

<img width="1299" height="393" alt="Image" src="https://github.com/user-attachments/assets/6b992a2c-2c3e-4a2a-9f7b-1264c2ff111d" />

Reply:
I can take care of it. I think it's a missed path when I worked the `wpcomLink()` links. There are more files with this path, so I'll fix them all.